docs/auth0_api.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+---
+## auth0 api
+
+Makes an authenticated HTTP request to the Auth0 Management API
+
+### Synopsis
+
+Makes an authenticated HTTP request to the Auth0 Management API and prints the response as JSON.
+
+The method argument is optional, and when you don’t specify it, the command defaults to GET for requests without data and POST for requests with data.
+
+Auth0 Management API Docs:
+  https://auth0.com/docs/api/management/v2
+
+Available Methods:
+  GET, POST, PUT, PATCH, DELETE
```

Comment:
Should these show up in the documentation/examples/errors exactly like they should be fed to the CLI, that is, lowercased? Otherwise, it seems to suggest that using them uppercased is the way to go.

Reply:
It doesn't really matter how these are passed through, if lowercase or uppercase: https://github.com/auth0/auth0-cli/pull/531/files#diff-9641610d0216306cf67e4b06eecff4cc3ec029d1bdbe4b53a6371b20ccdd9846R188. It felt a bit more elegant to present the uppercase ones in the help, but also show that they can be lowercase in the examples. Should be only allow lowercase or uppercase?

Reply:
I think having them be case-insensitive is fine. But I'd have them be lowercased in the help, as otherwise it may lead users to think that they _must_ be uppercased, which is slightly more of an inconvenience to type, specially when the rest of the command is lowercase.

Reply:
Updated in: [`b56f661` (#531)](https://github.com/auth0/auth0-cli/pull/531/commits/b56f66112567730d35fb9f8055872f15c704d987).